example-app/src/main/java/com/hcaptcha/example/MainActivity.java

```diff
@@ -68,6 +68,12 @@ private HCaptchaConfig getConfig() {
                 .hideDialog(hideDialog.isChecked())
                 .tokenExpiration(10)
                 .diagnosticLog(true)
+                .retryPredicate(new IHCaptchaRetryPredicate() {
+                    @Override
+                    public boolean shouldRetry(HCaptchaConfig config, HCaptchaException exception) {
+                        return exception.getHCaptchaError() == HCaptchaError.SESSION_TIMEOUT;
+                    }
+                })
```

Comment:
Should we replace it with lambda, like this?
```suggestion
                .retryPredicate((config, exception) -> exception.getHCaptchaError() == HCaptchaError.SESSION_TIMEOUT)
```

Reply:
Pushed https://github.com/hCaptcha/hcaptcha-android-sdk/pull/87/commits/7a5522169c975406a2a84c63b754b284fc348602